app/views/evaluations/export.ctp

```diff
@@ -64,6 +64,13 @@ echo $this->Form->input('student_id', array(
 ?>
 <div class='error-message' id='greenError'><?php echo __('Please include at least one of the green fields', true); ?></div>
 <?php
+echo $this->Form->input('question_title', array(
+  'type' => 'checkbox',
+  'name' => 'include[question_title]',
+  'checked' => true,
```

Comment:
Could you make this default unchecked? To be consistent with before.

Reply:
Done.